src/aws-ecr/orb.yml

```diff
@@ -0,0 +1,125 @@
+version: 2.1
+
+description: "An orb for working with Amazon Elastic Container Registry (ECR)
+              Sample usage within config.yml:
+
+              workflows:
+                build_and_push_image:
+                  jobs:
+                    - aws-ecr/build_and_push_image:
+                        context: myContext
+                        region: us-west-2
+                        account-url: myAwsAccountNum.dkr.ecr.myRegion.amazonaws.com 
+                        repo: myRepo
+                        tag: latest "
+orbs:
+  aws-cli: circleci/aws-cli@volatile
+
+executors:
+  default:
+    machine:
```

Comment:
I suspect we should use remote docker inside some basic docker image (TODO: we don't seem to have a "generic" convenience image, which would be good for this purpose).

Reply:
I'll test this out with setup_remote_docker and commit a new executor. Hold for that.